### DISCUSSION TITLE
2 , 3 idee en vrac

### DISCUSSION THREAD

**Discussion by Archimageifrit** [Crownicles/Crownicles#1106] [Suggestions fermées]
J'ai 2 , 3 idée en vrac qui on absolument aucun lien entre elles mais sympas quand même  

La première sa serait une sorte de nouvelles monnaie , ou à la place de revendre notre équipement on a le choix de le "brisé" pour obtenir des fragments qui permettront pourquoi pas d'upgrade nôtre équipement  ou d'acheter d'autres équipements dans une sorte de boutique ou on pourrait craft des item spéciaux 

La seconde sera que no pet puisse nous aidé un peu en combats , ai différente idée de comment donc en voilà qu'elle que une  :

Par un bonus pacife comme un boost de stats ( exemple bonus de force avec le loup ) ou 
d'autres effect avec des meilleures pet ( comme un regen avec la licorne ) 

Une attaque avec le pet qui serait plus ou moins puissant selon le pet et son bonheur ( se qui raconteras un intérêt supplémentaire à up leur moral ) 
 
Troisième idée sera de rajouter un peu de magie dans ce monde de brut . Avec un moyen quelconque de loot des item rare : les parchemins ! Qui aurons des effet pour certain quotidien avec le !daily ou en combat avec une attaque en plus ( avoué pouvoir lancé des boule de feu en combats  sa donne en vie ! ) ou d'autres parchemin plus utilitaires comme certains évitant le gel ou la maladie !

**Comment by Stitch-seagull** [comment]
Pour ce qui est de la seconde proposition, celle des pets, personnellement je verrais bien quelque chose du genre : 
- Le joueur choisit son attaque 
- Son attaque passe/passe pas
- Ensuite le pet a une chance aléatoire d'appliquer un effet de dégâts ou de régénération

**Reply by Archimageifrit** [reply]
Oui c'est une idée parmi tant d'autres , si l'idée plaît au dev à voir comment il limplemente

**Comment by BastLast** [comment]
Hello, il vaut mieux ouvrir une discussion par idée, ça facilite grandement le traitement pour nous :) je ferme cette discussion mais je t'invite à ouvrir de nouvelles discussions :)
